schema/Makefile

```diff
@@ -16,3 +16,6 @@ validate: validate.go
 	go get -d ./...
 	go build ./validate.go
 
+clean:
```

Comment:
If you do not add `.PHONY`, `make validate` can not be repeated, the following error occurs：
`make: 'validate' is up to date.`
so add the `.PHONY`.